### PR TITLE
Dev v1 Comparing changes to Main

### PR DESCRIPTION
**Pull request by Cat-Vader**
Lets clear this ASAP @kechykechy @cjephuneh @amosmachora

### CODE REVIEW

File: app/dashboard/page.tsx [hunk 1]
```diff
@@ -1,15 +1,54 @@
 "use client";
 
-import { IMsalContext, MsalAuthenticationTemplate } from "@azure/msal-react";
-import { faSpinner } from "@fortawesome/free-solid-svg-icons";
-import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
-import React, { ElementType } from "react";
```
Comment (amosmachora) [inline app/dashboard/page.tsx:6]: Resolve merge conflicts in this file

**Review by amosmachora — CHANGES_REQUESTED** [review]
There are a bunch of merge conflicts with the main branch. Fix them and then we will try to merge

File: app/dashboard/page.tsx (hunk 1 above)
Comment (cjephuneh) [reply app/dashboard/page.tsx:6]: okay on it

**Comment by cjephuneh** [comment]
i am done with this

**Review by cjephuneh — APPROVED** [review]

**Review by cjephuneh — APPROVED** [review]

**Comment by cjephuneh** [comment]
azure piplines????

**Comment by Cat-Vader** [comment]
Yes, for CI/CD. Access the logs for anypart the build breaks
________________________________

**Pull request closed without merging**